requirements.txt

```diff
@@ -1,3 +1,3 @@
 numpy>=1.12.0
-niworkflows>=0.2.7
+git+https://github.com/chrisfilo/niworkflows.git@21c0647bb7e42938e947334ab8c474c011fe3484#egg=niworkflows-0.3.1-dev
```

Comment:
I guess you are planning on merging poldracklab/niworkflows#221 and release first, so the requirements will finally be pinned to that new release, right?

Reply:
Yes - that's why this PR is still marked as WIP.